examples/gke-a4x/README.md

```diff
@@ -0,0 +1,49 @@
+## Deployment Instructions
+
+1. Create a Google Cloud Workstation.
+1. Clone the Cluster Toolkit repository.
```

Comment:
Modify README by looking at [A3 mega cloud doc](https://cloud.google.com/cluster-toolkit/docs/deploy/deploy-a3-mega-gke-cluster) This is what customers will observe.